Modules/_ctypes/ctypes.h

```diff
@@ -47,6 +48,13 @@ typedef struct {
     PyTypeObject *PyCArrayType_Type;
     PyTypeObject *PyCSimpleType_Type;
     PyTypeObject *PyCFuncPtrType_Type;
+    PyTypeObject *PyCData_Type;
+    PyTypeObject *Struct_Type;
+    PyTypeObject *Union_Type;
+    PyTypeObject *PyCArray_Type;
+    PyTypeObject *Simple_Type;
+    PyTypeObject *PyCPointer_Type;
```

Comment:
These fields are ordered by their declaration order in `_ctypes.c`, so the `PyComError_Type` should be at this position. Although it's not added in this PR, but I think it better to fixed instead of a seprated PR.

Reply:
Done. Am I following you?

Reply:
Yes, exactly!